[EdgeTPU] Update the testing codes to register EdgeTPUToolchain

rhqjatn2398 opened the pull request:
This PR modifies the testing codes to test registration of EdgeTPUToolchain.
Also, This commit exports BackendMap interface to use it in the testing codes.

From #73 

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

rhqjatn2398 committed:
[EdgeTPU] Update the testing codes to register EdgeTPUToolchain

This commit modify the testing codes to test registration of EdgeTPUToolchain.
Also, This commit exports BackendMap interface to use it in the testing codes.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

rhqjatn2398 committed:
[EdgeTPU] Fix the testing codes

This commit fixes the 'registers a backend' testing codes.

After the calling of API.registerBackend, it adds the element to gToolchainEnvMap and globalBackendMap.
So globalBackendMap should be cleaned up.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

rhqjatn2398 committed:
[EdgeTPU] Update the testing codes

This commit moves the teardown function to the bottom of the test suite.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

profornnan commented on src/Tests/Backend/Backend.test.ts:
https://github.com/SSDC-onE-FiVE/ONE-vscode/blob/78db744365d47ffe72513de98151abc0cf7753ae/src/Tests/Backend/Backend.test.ts#L87-L95

Could you please remove this teardown? I added it but we run test code based on backend that registered in extension.ts 😭

rhqjatn2398 replied:
I got it.
I'll remove that. 🖊️

rhqjatn2398 committed:
[EdgeTPU] Remove the deletetion codes

This commit remove the teardown function.


ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

profornnan reviewed (approved):
Thank you~!! LGTM 👍
